bedrock/firefox/templates/firefox/campaign/unfck/index.fr.html

```diff
@@ -0,0 +1,160 @@
+        link_text='Découvrir comment Firefox vous protège',
+        link_url='https://blog.mozilla.org/firefox/fr/firefox-vous-montre-qui-vous-suit-autour-du-web/',
+        id='protect',
+        src_name='cat',
+        alt_text='Un chat rusé attrape ses profils cibles avec sa langue.',
+        share_text='Partager',
+        download_text='Télécharger le GIF',
+        copy_text='Copier le lien du GIF',
+        copy_success_text='Lien copié'
+      ) %}{% endcall %}
+
+      {% call share_item(
+        title='N’acceptez pas de faire parti d’un système',
+        info='La technologie qui nous rassemble nous monétise et nous manipule aussi. Il vous suffit de regarder le documentaire "The Social Dilemma" avec nos amis du “Center for Human Tech” pour comprendre comment le Net et les milliards de personnes qui l’utilisent sont devenus des marchandises pour les géants de la Tech.',
+        link_text='En savoir plus sur le documentaire.',
+        link_url='https://blog.mozilla.org/firefox/fr/derriere-nos-ecrans-de-fumee',
+        id='social',
+        src_name='button-fr',
+        alt_text='La souris “Fck” clique sur « Je n’accepte pas »'
+      ) %}{% endcall %}
+
+      {% call share_item(
+        title='N’acceptez pas les intox',
+        info='Qu’il s’agisse de mensonges, de fake news ou de bons vieux canulars, la désinformation se répand plus rapidement que jamais. Grâce à quelques conseils et des outils simples, vous pouvez vous en protéger ainsi que les autres.',
+        link_text='En savoir plus',
+        link_url='https://blog.mozilla.org/firefox/fr/comment-eviter-la-desinformation-en-ligne/',
+        id='misinformation',
+        src_name='purrfect',
+        alt_text='La patte d’un chat chasse les intox pour révéler les vrais infos.',
+        share_text='Partager',
+        download_text='Télécharger le GIF',
+        copy_text='Copier le lien du GIF',
+        copy_success_text='Lien copié'
+      ) %}{% endcall %}
+
+      {% call share_item(
+        title='Acceptez de faire parti du mouvement #pourunnetplusnet',
+        info='Parlons librement de ce qui n’est plus très net sur le Net. Plus les gens en parleront, plus on y verra clair.',
+        id='share',
+        src_name='one-fck-given',
+        alt_text='Deux mains se font un high five célébrant le net plus net.',
+        include_cta=True,
+        share_text='Partager',
+        download_text='Télécharger le GIF',
+        copy_text='Copier le lien du GIF',
+        copy_success_text='Lien copié'
+        ) %}
+        <form class="c-tell" action="{{ 'https://www.twitter.com/intent/tweet?url=%s&hashtags=%s&via=%s'|format(tw_url, tw_hashtags, tw_via)|e }}" method="post">
+          <textarea class="c-tell-input" placeholder="vous y voyez encore clair sur le Net ?" required maxlength="280"></textarea>
+          <button type="submit" class="c-tell-button mzp-c-button mzp-t-secondary js-tell-button">Partager</button>
```

Comment:
Looks like this is using the DE short link and Twitter handle:

![Screen Shot 2020-10-05 at 14 33 09](https://user-images.githubusercontent.com/854701/95134435-b9eea880-0717-11eb-9430-a1f770ba0286.png)